app-android/src/main/java/com/ustadmobile/port/android/view/binding/ImageViewBindings.kt

```diff
@@ -262,6 +261,36 @@ private fun ImageView.updateFromImageLookupMap() {
     }
 }
 
+@BindingAdapter("isContentCompleteImage")
+fun ImageView.isContentCompleteImage(person: PersonWithSessionsDisplay){
+    if(person.resultComplete){
+        when(person.resultSuccess){
+            StatementEntity.RESULT_SUCCESS -> {
+                setImageResource(R.drawable.exo_ic_check)
+                visibility = View.VISIBLE
+            }
+            StatementEntity.RESULT_FAILURE -> {
+                setImageResource(R.drawable.ic_close_black_24dp)
+                visibility = View.VISIBLE
+            }
+            StatementEntity.RESULT_UNSET ->{
+                setImageDrawable(null)
+                visibility = View.INVISIBLE
+            }
+        }
+    }else{
+        setImageDrawable(null)
+        context.getString(R.string.incomplete)
+        visibility = View.INVISIBLE
+    }
+}
+
+@BindingAdapter("verbIcon")
+fun ImageView.setVerbIcon(verb: String?) {
```

Comment:
We already have imageLookupKey and imageLookupMap view bindings (which is a generic handler for setting an image based on some key in a map). It is documented. We don't need an extra view bindings just for verbs. You can use the verb's hardcoded uid constant instead of the url.